sim/core/unit.go

```diff
@@ -428,7 +428,7 @@ func (unit *Unit) MoveTo(moveRange float64, sim *Simulation) {
 	unit.moveSpell.Cast(sim, unit.CurrentTarget)
 
 	sim.AddPendingAction(NewPeriodicAction(sim, PeriodicActionOptions{
-		Period:          time.Millisecond * 500,
+		Period:          time.Millisecond * 143,
```

Comment:
Why 143ms? That's an oddly specific number 😄

Reply:
Time per yd moved. I suppose I could've written 1000 / 7 there to be accurate, since it's 7 yd/s.

Reply:
Fixed this